syntax error: unexpected [ after top level declaration

The package fails to build with go1.13.8 linux/amd64

```
# github.com/urfave/cli
../go/src/github.com/urfave/cli/flag_bool.go:9:25: syntax error: unexpected [ after top level declaration
```

--- a/.gitignore
+++ b/.gitignore
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+/git-rm-merged
 
 # Test binary, build with `go test -c`
 *.test

--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,4 @@ module github.com/yasukotelin/git-rm-merged
 
 go 1.13
 
-require github.com/urfave/cli v1.22.1
+require github.com/urfave/cli v1.22.10

